modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotLocalFileStorage.scala

```diff
@@ -0,0 +1,31 @@
+package org.tessellation.infrastructure.snapshot
+
+import cats.Applicative
+import cats.data.EitherT
+import cats.effect.Async
+import cats.syntax.flatMap._
+
+import org.tessellation.dag.snapshot.{GlobalSnapshot, SnapshotOrdinal}
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.storage.LocalFileStorage
+
+import fs2.io.file.Path
+
+final class GlobalSnapshotLocalFileStorage[F[_]: Async: KryoSerializer] private (path: Path)
```

Comment:
I would rename it to `GlobalSnapshotLocalFileSystemStorage` or `GlobalSnapshotLocalFSStorage` because currently, someone could think that the whole storage is in a single file, like DB file (while it's in the directory).